python/observability-online-eval/coding-agent/coding_agent/prompts/file_format.md

```
@@ -0,0 +1,25 @@
+## Output Format
+
+You will output the content of each file necessary to achieve the goal, including ALL code.
+Represent files like so:
+
+FILENAME
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)